pkgs/development/python-modules/i3ipc/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, buildPythonPackage, enum-compat }:
+
+let
+  version = "1.3.0";
+  sha256 = "1rw9mq18np6bf8xp8vvc21bi5q8xjmj8dck2vsa1hy8bk434259m";
+
+in buildPythonPackage rec {
+  name = "i3ipc-${version}";
+  inherit version;
+
+  src = pkgs.fetchFromGitHub {
+    owner = "acrisci";
+    repo = "i3ipc-python";
+    rev = "refs/tags/v${version}";
+    inherit sha256;
+    fetchSubmodules = true; # because a tag is used
+  };
+
+  doCheck = false;
```

Comment:
Why? Leave a comment in the code with a reason.

Reply:
Added a comment